### PR TITLE
Use correct path in example LoadFile 

### LINKED ISSUE #1163: CI: `example_load_file.py` isnt running
**Describe the bug**
 The `python-sdk/example_dags/example_load_file.py` example DAG in not running in our CI tests

**Version**
* Astro: [e.g. 0.6.0]
* OS: [eg. Debian]

**To Reproduce**
1. Write the DAG  '...'
2. Create connection '....'
3. Run using  '....'

**Expected behavior**

**Screenshots**

**Additional context**

### PR DESCRIPTION
**Pull request by pankajastro**
# Description
closes #1163 
## What is the current behavior?
Example Loadfile uses a resource which does not exist.

Since the file does not exist load_file_to_table_natively is failing and it falls for the pandas path. In the pandas path the resolve_file_path_pattern converting path pattern to gs://astro-sdk/workspace/sample_pattern.csv and luckily we do have this file in GCS.

## What is the new behavior?
changes the resource path

## Does this introduce a breaking change?
No

### Checklist
- [ ] Created tests which fail without the change (if possible)
- [ ] Extended the README / documentation, if necessary

### CODE REVIEW

**Comment by pankajastro** [comment]
Since the file does not exist load_file_to_table_natively is failing and it falls for the pandas path. In the pandas path the resolve_file_path_pattern converting path pattern to gs://astro-sdk/workspace/sample_pattern.csv and luckily we do have this file in GCS.

Why extractor tests is failing? since I'm my PR we have changed the enable native fallback behaviour

**Review by phanikumv — APPROVED** [review]

**Comment by utkarsharma2** [comment]
@pankajastro The reason it is failing is that earlier we were using https://github.com/apache/airflow/blob/eb47c42d6ba3ca33cf4223ac6c2a4904cf1f388e/airflow/providers/google/cloud/hooks/gcs.py#L678 method for pattern resolution which expects a prefix. Now we have changed this behavior to https://github.com/apache/airflow/blob/eb47c42d6ba3ca33cf4223ac6c2a4904cf1f388e/airflow/providers/google/cloud/hooks/bigquery.py#L1481
as bigquery has the ability to process pattern natively. 

PR - https://github.com/astronomer/astro-sdk/commit/ae9ff4aa57dd02cbf4fb5a5f2bb9490190e1079f

Also, it will work if we add `gs://astro-sdk/workspace/sample_pattern*`

Can you please update this is the PR?

File: python-sdk/example_dags/example_load_file.py [hunk 1]
```diff
@@ -142,7 +142,7 @@
     # [START load_file_example_12]
     aql.load_file(
         input_file=File(
-            "gs://astro-sdk/workspace/sample_pattern",
+            "gs://astro-sdk/workspace/sample_pattern.csv",
```
Comment (pankajastro) [inline python-sdk/example_dags/example_load_file.py:145]:
```suggestion
            "gs://astro-sdk/workspace/sample_pattern*",
```

File: python-sdk/example_dags/example_load_file.py [hunk 2]
```diff
@@ -166,7 +166,7 @@
     # [START load_file_example_14]
     aql.load_file(
         input_file=File(
-            "gs://astro-sdk/workspace/sample_pattern",
+            "gs://astro-sdk/workspace/sample_pattern.csv",
```
Comment (pankajastro) [inline python-sdk/example_dags/example_load_file.py:169]:
```suggestion
            "gs://astro-sdk/workspace/sample_pattern*",
```

File: python-sdk/example_dags/example_load_file.py [hunk 3]
```diff
@@ -200,7 +200,7 @@
     # [START load_file_example_17]
     aql.load_file(
         input_file=File(
-            "gs://astro-sdk/workspace/sample_pattern",
+            "gs://astro-sdk/workspace/sample_pattern.csv",
```
Comment (pankajastro) [inline python-sdk/example_dags/example_load_file.py:203]:
```suggestion
            "gs://astro-sdk/workspace/sample_pattern*",
```

**Comment by pankajastro** [comment]
> @pankajastro The reason it is failing is that earlier we were using https://github.com/apache/airflow/blob/eb47c42d6ba3ca33cf4223ac6c2a4904cf1f388e/airflow/providers/google/cloud/hooks/gcs.py#L678 method for pattern resolution which expects a prefix. Now we have changed this behavior to https://github.com/apache/airflow/blob/eb47c42d6ba3ca33cf4223ac6c2a4904cf1f388e/airflow/providers/google/cloud/hooks/bigquery.py#L1481 as bigquery has the ability to process patterns natively, but now we don't expect a prefix.
> 
> PR - [ae9ff4a](https://github.com/astronomer/astro-sdk/commit/ae9ff4aa57dd02cbf4fb5a5f2bb9490190e1079f)
> 
> Also, it will work if we add `gs://astro-sdk/workspace/sample_pattern*`
> 
> Can you please update this is the PR?
Look like it does not work for airflow-2-2-5 https://github.com/astronomer/astro-sdk/actions/runs/3384140248/jobs/5620762018

File: python-sdk/example_dags/example_load_file.py [hunk 4]
```diff
@@ -142,7 +142,7 @@
     # [START load_file_example_12]
     aql.load_file(
         input_file=File(
-            "gs://astro-sdk/workspace/sample_pattern",
+            "gs://astro-sdk/workspace/sample_pattern*",
```
Comment (pankajastro) [inline python-sdk/example_dags/example_load_file.py:145]:
```suggestion
            "gs://astro-sdk/workspace/sample_pattern.csv",
```

File: python-sdk/example_dags/example_load_file.py [hunk 5]
```diff
@@ -166,7 +166,7 @@
     # [START load_file_example_14]
     aql.load_file(
         input_file=File(
-            "gs://astro-sdk/workspace/sample_pattern",
+            "gs://astro-sdk/workspace/sample_pattern*",
```
Comment (pankajastro) [inline python-sdk/example_dags/example_load_file.py:169]:
```suggestion
            "gs://astro-sdk/workspace/sample_pattern.csv",
```

File: python-sdk/example_dags/example_load_file.py [hunk 6]
```diff
@@ -200,7 +200,7 @@
     # [START load_file_example_17]
     aql.load_file(
         input_file=File(
-            "gs://astro-sdk/workspace/sample_pattern",
+            "gs://astro-sdk/workspace/sample_pattern*",
```
Comment (pankajastro) [inline python-sdk/example_dags/example_load_file.py:203]:
```suggestion
            "gs://astro-sdk/workspace/sample_pattern.csv",
```

**Pull request merged**